SgfDevs/Views/Shared/_Layout.cshtml

```diff
@@ -69,8 +69,8 @@
                     @*<li><a href="/events/">Events</a></li>*@
                     <li><a href="/groups">Groups</a></li>
                     <li><a href="/directory">Directory</a></li>
-                    <li><a href="/jobs">Jobs</a></li>
-                    <li><a href="/archives">Archives</a></li>
+                    @* <li><a href="/jobs">Jobs</a></li> *@
+                    <li><a href="https://youtube.com/playlist?list=PLsuKlCbZhK0Of5hMBU7CZeunPHIciRfPD&si=Iv9RVVeOHaCAexIc">Archives</a></li>
```

Comment:
From what I can tell most of the external links on the site currently open in a new tab. Can you add that to this one too?